FirstProject/css/main.css

```diff
@@ -0,0 +1,149 @@
+
+/*Header section*/
+header {
+    background-image: url(../img/feedback-banner.jpg);
+    width: 100%;
+}
+
+.left_image {
```

Comment:
Any particular reason for using `snake_case` here, and `spinal-case` below? Can we settle on one, for consistency?